backend/prisma/README.md

```diff
@@ -1,6 +1,11 @@
-https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch/relational-databases/connect-your-database-typescript-postgres
+Prefix all commands to access the correct .env file with: dotenv -e ../.env
+
+Connecting DB: https://www.prisma.io/docs/getting-started/setup-prisma/start-from-scratch/relational-databases/connect-your-database-typescript-postgres
+- 1. Replace the database connection string in the schema.prisma with your own.
+
+DB Migration: https://www.prisma.io/docs/concepts/components/prisma-migrate/get-started
+- 1. make sure to install the dotenv dependency: cd backend npm install -g dotenv-cli 
```

Comment:
fyi: the `-g` flag here means global, so it actually doesn't matter what directory you're in when you run this command! so we can remove the `cd backend` step

Reply:
okay i will revise that